faunadb/json.go

```diff
@@ -201,20 +201,29 @@ func (p *jsonParser) parseQuery() (value Value, err error) {
 	var lambda json.RawMessage
 
 	if err = p.decoder.Decode(&lambda); err == nil {
-		value = QueryV{lambda}
+		value = QueryV{lambda: lambda}
 	}
 
 	var token json.Token
 	if token, err = p.decoder.Token(); err == nil {
-		if token != json.Delim('}') {
-			err = wrongToken{"end of object", token}
+		switch token {
+		case "api_version":
```

Comment:
not sure if it will work because `api_version` is not guaranteed to be the last field, it can be in any order

Reply:
Marrony is right. We can't assume predictable order.

Reply:
actually, most of the local queries, it's returning on the first field:
```
curl -H "X-FaunaDB-API-Version: 3" -u secret: localhost:8443/ --data '{"@query":{"lambda":"x","expr":{"var":"x"},"api_version": "3"}}'
{
  "resource": {
    "@query": {
      "api_version": "3",
      "lambda": "x",
      "expr": { "var": "x" }
    }
  }
}
```